locale/messages.pot

```diff
@@ -243,108 +243,120 @@ msgstr ""
 msgid "TOGGLE_ENGULF"
 msgstr ""
 
-#: ../simulation_parameters/common/input_options.json:51
+#: ../simulation_parameters/common/input_options.json:46
+msgid "TOGGLE_BINDING"
+msgstr ""
+
+#: ../simulation_parameters/common/input_options.json:50
+msgid "TOGGLE_UNBINDING"
```

Comment:
Is this still used? Did you implement both "unbind all" and "unbind specific" cell from colony buttons? If both of those don't exist, then one of these text translations is probably extraneous.

Reply:
No I didn't, but there are three new inputs
- Toggle binding mode (B)
- Toggle unbinding mode (U)
- Accept unbind (Left click)

I have seperate toggle unbinding mode and accept unbind logic. This is due to the cells rotating in a colony and the user therefore unable to click on a cell. Pressing U flashes the colony orange and you can't move anymore.

Reply:
So there isn't a quick, unbind everything button? I think that's a more common requirement as I imagine the player does this:
- swim around
- bind to other cells
- collect enough stuff to reproduce
- ditch the entire colony with a single button press
- go to the editor

so making that easy should be a priority.

Reply:
I mean, this would add another button. I think this would slowly become too many inputs, because the player could also just press U and click on the player cell.
Another possibillity would be to add an input for ditching the colony, but to not assign a default button to it. Advanced users can assign this input.

Reply:
I'd prefer removing the manual unbinding mode entirely, then. I think it's unnecessary, and making it work for controller input would be quite a bit of work. As I outlined in that game flow, it's much more important to just quickly ditch your colony than remove individual members.

Reply:
I think I made a good compromise with:
U - Unbind all
Shift+U - Go to unbind mode to unbind single cells

Reply:
I guess that could work, but no player will be able to discover that without a tutorial. I think a better choice is to have separate keys for those actions, then the player at least can find those in the input bindings menu.

Reply:
They can see all keybindings in the menu.

Reply:
![image](https://user-images.githubusercontent.com/30809803/113983526-49515180-984a-11eb-81c1-5c1aebe31f1c.png)

Reply:
Ah, yes.